oat-sa/extension-tao-mediamanager
# Feature/AUT-18/Style editor - text block in passage

Related to: https://oat-sa.atlassian.net/browse/AUT-18

Added style editor for text blocks.

How to test:
- go to Assets
- create new Passage
- edit passage - add text blocks, set different styles for passage and for block
- save passage
- check preview
- close passage
- open again
- check all styles applied
